apps/products-feed/src/pages/index.tsx

```diff
@@ -18,14 +18,16 @@ const IndexPage: NextPage = () => {
   }, [isMounted, appBridgeState?.ready]);
 
   if (isInIframe()) {
-    return <LinearProgress />;
+    return <Text color={"textNeutralSubdued"}>Loading...</Text>;
   }
 
   return (
     <div>
-      <h1>Saleor Product Feed</h1>
-      <p>This is Saleor App that allows product feed generation</p>
-      <p>Install app in your Saleor instance and open in with Dashboard</p>
+      <Text variant={"hero"} as={"h1"} marginBottom={8}>
+        Saleor Product Feed
```

Comment:
```suggestion
        Saleor Products Feed
```

Reply:
Actually is `product` not `products` feed ;)

Reply:
its only folder :) will do eventually